manipulation/src/main/java/org/jboss/gm/manipulation/actions/AlignedDependencyResolver.java

```diff
@@ -43,6 +49,11 @@ public AlignedDependencyResolver(ManipulationModel module) {
     public void execute(DependencyResolveDetails resolveDetails) {
         if (!BYPASS) {
             final ModuleVersionSelector requested = resolveDetails.getRequested();
+            if (isEmpty(requested.getVersion())) {
```

Comment:
Is this a similar problem to the getAllProjectDependencies which also needed a similar check ?

Reply:
I think you run into this yesterday - this is to avoid "managed dependencies". The logic bellow fails when the version is empty.

Reply:
Will add comment.

Reply:
Ok thanks